### PR TITLE
Add PR tester jenkinsfile

### PR DESCRIPTION
**Pull request by msajidmansoori12**
Add jenkins pipeline to test PRs again tier3 tests (JIRA tests) which need some sensitive data.

### CODE REVIEW

File: PR_TESTER.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                script{
+                    env.MINIKUBE_IP = sh(
+                        script : 'minikube ip',
+                        returnStdout: true
+                    ).trim()
+                    env.HUB_BASE_URL="http://"+env.MINIKUBE_IP+"/hub"
+                    env.TACKLE_API_URL = env.MINIKUBE_IP+":443"
+                    sh""" openssl s_client -showcerts -connect $TACKLE_API_URL </dev/null | sed -n '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > chain.pem
+                          sudo cp chain.pem /etc/pki/ca-trust/source/anchors/
+                          sudo update-ca-trust"""
```
Comment (mguetta1) [inline PR_TESTER.jenkinsfile:85]: no need to import the certificate chain for "http"

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by mguetta1 — APPROVED** [review]
LGTM

File: PR_TESTER.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    agent {
+        label 'minikube-node' 
+    }
+    options{
+        ansiColor('xterm')
+    } 
+    environment{
+        HUB_USERNAME="admin"
+        PATH="$PATH:/usr/local/go/bin"
+        HUB_PASSWORD="Passw0rd!"
```
Comment (aufi) [inline PR_TESTER.jenkinsfile:12]: Assuming this is some example password, not working on some running system.

**Review by aufi — COMMENTED** [review]
I'm not very familiar with Jenkins, but looks good.

File: PR_TESTER.jenkinsfile (hunk 2 above)
Comment (mguetta1) [reply PR_TESTER.jenkinsfile:12]: It is the default password and it is used with `keycloak_sso_req_passwd_update: false`

Comment (aufi) [reply PR_TESTER.jenkinsfile:12]: I don't think it is the right thing to store even default passwords in public git repo, but just my $0.02.

Comment (msajidmansoori12) [reply PR_TESTER.jenkinsfile:12]: @aufi Thanks for your input , I don't mind removing these from here. I'll make the changes :)

**Comment by msajidmansoori12** [comment]
@aufi @mguetta1 i've made the final changes , can you please merge this PR.

**Pull request merged**